tests/nnstreamer_filter_reload/tensor_filter_reload_test.c

```diff
@@ -41,8 +40,8 @@ static gint return_val = 0;
 static gboolean
 bus_callback (GstBus * bus, GstMessage * message, gpointer data)
 {
+  GMainLoop *loop = data;
```

Comment:
you should add user_data when adding watch callback in main function,
`gst_bus_add_watch (bus, bus_callback, loop);`

Reply:
thanks! fixed.